azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -340,8 +340,13 @@ public Job buildJobExecutor(String jobId, Props jobProps, Logger logger)
             jobProps, jobType));
       }
 
-      // TODO: should the logic below mirror the logic for PluginLoadProps?
       Props pluginJobProps = pluginSet.getPluginJobProps(jobType);
+      // For default jobtypes, even though they don't have pluginJobProps configured,
+      // they still need to load properties from common.properties file if it's present
+      // because common.properties file is global to all jobtypes.
+      if(pluginJobProps == null) {
+        pluginJobProps = pluginSet.getCommonPluginJobProps();
+      }
       if (pluginJobProps != null) {
```

Comment:
Another thing is that custom jobtype will still read jobprops defined in azkaban-plugin, right? You might want to emphasize it here in comments.